paddle/fluid/inference/tests/api/preprocess.py

```diff
@@ -0,0 +1,109 @@
+#   copyright (c) 2019 paddlepaddle authors. all rights reserved.
+#
+# licensed under the apache license, version 2.0 (the "license");
+# you may not use this file except in compliance with the license.
+# you may obtain a copy of the license at
+#
+#     http://www.apache.org/licenses/license-2.0
+#
+# unless required by applicable law or agreed to in writing, software
+# distributed under the license is distributed on an "as is" basis,
+# without warranties or conditions of any kind, either express or implied.
+# see the license for the specific language governing permissions and
+# limitations under the license.
+import unittest
+import os
+import numpy as np
+import time
+import sys
+import random
+import functools
+import contextlib
+from PIL import Image, ImageEnhance
+import math
+
+random.seed(0)
+np.random.seed(0)
+
+DATA_DIM = 224
+
+SIZE_FLOAT32 = 4
+SIZE_INT64 = 8
+
+DATA_DIR = '/data/ILSVRC2012'
```

Comment:
If I run `cd build; python ../paddle/fluid/inference/tests/api/preprocess.py`, where will the output data?

Reply:
Currently it outputs to /data/ILSVRC2012/data.bin.  I also wanted to ask this. Where should I put the output data.bin

Reply:
`/data/ILSVRC2012/data.bin` is not better, since we don't have `/data` authority. `./data/ILSVRC2012/data.bin`?

Reply:
@lidanqing-intel you can put in the .cache like V1 did :https://github.com/PaddlePaddle/Paddle/blob/2632327429ed823cf1a0c2593cfa411fb2f111b9/python/paddle/fluid/contrib/tests/test_calibration.py#L120

Reply:
I think `.cache` is OK.

Reply:
ok will use .cache

Reply:
@luotao1  the data.bin path will be `~/.cache/int8_full_val_bin/data.bin` Is it ok ? I am worried if I put directly in  `~/.cache/` may cause some misunderstanding

Reply:
~/.cache/int8_full_val_bin/data.bin is OK

Reply:
Since V1 put into `~/.cache/paddle/dataset/int8/download`, how about put into `~/.cache/paddle/dataset/int8/download/int8_full_val.bin`

Reply:
> Since V1 put into `~/.cache/paddle/dataset/int8/download`, how about put into `~/.cache/paddle/dataset/int8/download/int8_full_val.bin`

Yes! Agree.

Reply:
I see the unzip dataset is not in this location.